code/src/Components/Form.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+
+import { Welcome } from "./Welcome";
+import { TextInput } from "./TextInput"
+import { CheckboxInput } from "./CheckboxInput";
+import { SelectInput } from "./SelectInput";
+import { RadioInput } from "./RadioInput";
+import { EmailInput } from "./EmailInput";
+import { Summary } from "./Summary";
+
+// ALl use state and event handlers for the page are here
+
+export const Form = () => {
+  const [count, setCount] = useState(1);
+  const [name, SetName] = useState("");
+  const [membership, SetMembership] = useState(false);
+  const [location, SetLocation] = useState("");
+  const [occupationGroup, SetOccupationGroup] = useState();
+  const [email, setEmail] = useState("")
+
+  // a counter in order to render the different steps/compontents of site
+
+  const Counter = () => {
+    setCount(count + 1)
+  }
+
+  const handleNameInputChange = (event) => {
+    SetName(event.target.value);
+  }
+
+  const handleMembershipInputChange = (event) => {
+    SetMembership(event.target.value);
+  }
+
+  const handleLocationInputChange = (event) => {
+    SetLocation(event.target.value);
+  }
+
+  const handleOccupationGroupInputChange = (event) => {
+    SetOccupationGroup(event.target.value);
+  }
+
+  const handleEmailInputChange = (event) => {
+    setEmail(event.target.value);
+  }
+
+  // a conditional to render, in combination with the counter function, the compontents one at a time
+
+  if (count === 1) {
+    return (
+      <div className="form-container">
+      <Welcome/>
+      <button onClick={Counter}>START</button>
+      </div>
+    )
+  } else if (count === 2) {
+    return (
+      <div className="form-container">
+      <TextInput
+        name={name}
+        onNameInputChange={handleNameInputChange}
```

Comment:
If you start with return and then the conditional rendering I don't think you have to write "return" for each block. Or you can write: 
 ```
{step === 1 && (
        <component/>
      )}
```
This allows you to remove extra code and I believe it makes your code cleaner.